File: SGA_Api/SGA_Api/Models/Palet/CerrarPaletMobilityDto.cs
namespace SGA_Api.Models.Palet
{
    public class CerrarPaletMobilityDto
    {
        public int UsuarioId { get; set; }
        public string CodigoAlmacen { get; set; } = ""; // almacén origen
        public short CodigoEmpresa { get; set; }
        public string? Comentario { get; set; }
        public decimal? Altura { get; set; }
        public decimal? Peso { get; set; }
    }
} 

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/traspasos/CerrarPaletMobilityDto.kt
package com.example.sga.data.dto.traspasos

data class CerrarPaletMobilityDto(
    val usuarioId: Int,
    val codigoAlmacen: String,
    val codigoEmpresa: Short,
    val ubicacionOrigen: String? = null // ✅ OPCIONAL
)

